doc/usecases/kdb/UC_store_config.md

```diff
@@ -0,0 +1,27 @@
+# Use Case: Storing application configuration
+
+## Summary
+
+- **Scope:** `libelektra-kdb`
+- **Level:** Developer Goal
+- **Actors:** Application, Elektra
+- **Brief:** Application stores configuration into KDB
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:**
+  - Application asks to store new configuration into KDB
+  - Elektra stores in backends
```

Comment:
This is confusing, unclear what you want to say here.

```suggestion
  - Elektra passes `KeySet` to backends, responsible to set the configuration as requested
```

Reply:
"responsible to set the configuration as requested" is even more confusing IMO and provides basically zero information.

Actually there is just a word missing here:

```suggestion
  - Elektra stores configuration in backends
```